### PR TITLE
[Async Clipboard article] Document dealing with pasted files

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Document dealing with pasted files.

### CODE REVIEW

File: src/site/content/en/blog/async-clipboard/index.md [hunk 1]
```diff
@@ -209,6 +209,23 @@ async function getClipboardContents() {
 }
 ```
 
+### Working with pasted files
+
+It is useful for users to be able to use clipboard keyboard shortcuts such as
+<kbd>ctrl</kbd>+<kbd>c</kbd> and <kbd>ctrl</kbd>+<kbd>v</kbd>.
+Chromium exposes _read-only_ files on the clipboard as outlined below.
```
Comment (jpmedley) [inline src/site/content/en/blog/async-clipboard/index.md:216]: I don't think the code example illustrates the read-only nature of clipboard items. Specifically, if such items were read/write, this sample would still work.

File: src/site/content/en/blog/async-clipboard/index.md [hunk 2]
```diff
@@ -209,6 +209,23 @@ async function getClipboardContents() {
 }
 ```
 
+### Working with pasted files
```
Comment (jpmedley) [inline src/site/content/en/blog/async-clipboard/index.md:212]: Are you implying that simply implementing copy, cut, and pate events will make the keyboard shortcuts function? You should say that if such is the case.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/async-clipboard/index.md [hunk 3]
```diff
@@ -209,6 +209,23 @@ async function getClipboardContents() {
...
+
+It is useful for users to be able to use clipboard keyboard shortcuts such as
+<kbd>ctrl</kbd>+<kbd>c</kbd> and <kbd>ctrl</kbd>+<kbd>v</kbd>.
+Chromium exposes _read-only_ files on the clipboard as outlined below.
+
+```js
+document.addEventListener('paste', async (e) => {
+  if (!e.clipboardData.files) {
```
Comment (tomayac) [inline src/site/content/en/blog/async-clipboard/index.md:220]:
```suggestion
  if (!e.clipboardData.files || !e.clipboardData.files.length) {
```

File: src/site/content/en/blog/async-clipboard/index.md (hunk 2 above)
Comment (tomayac) [reply src/site/content/en/blog/async-clipboard/index.md:212]: I have clarified this.

File: src/site/content/en/blog/async-clipboard/index.md (hunk 1 above)
Comment (tomayac) [reply src/site/content/en/blog/async-clipboard/index.md:216]: I have added a comment that clarifies this.

**Comment by tomayac** [comment]
@rachelandrew, in Joe's absence, could you help land this, please?

File: src/site/content/en/blog/async-clipboard/index.md [hunk 4]
```diff
@@ -209,6 +209,28 @@ async function getClipboardContents() {
 }
 ```
 
+### Working with pasted files
+
+It is useful for users to be able to use clipboard keyboard shortcuts such as
+<kbd>ctrl</kbd>+<kbd>c</kbd> and <kbd>ctrl</kbd>+<kbd>v</kbd>.
+Chromium exposes _read-only_ files on the clipboard as outlined below.
+This triggers when the user hits the operating system's default paste shortcut
+or when the usere clicks the **Edit > Paste** menu in the browser's menu bar.
```
Comment (jpmedley) [inline src/site/content/en/blog/async-clipboard/index.md:218]:
```suggestion
or when the usere clicks the **Edit** then **Paste** in the browser's menu bar.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by Vlastimilstanik — APPROVED** [review]

**Review by mihajlija — APPROVED** [review]

**Pull request merged**